misk/src/main/kotlin/misk/web/interceptors/RequestBodyLoggingInterceptor.kt

```diff
@@ -90,9 +90,7 @@ internal class RequestResponseCapture @Inject constructor() {
     private val capture = ThreadLocal<RequestResponseBody>()
   }
 
-  fun get(): RequestResponseBody {
-    return capture.get()
-  }
+  fun get(): RequestResponseBody? = capture?.run { get() }
```

Comment:
capture is never null, but capture.get() can be. All you need is to change the output type.

```suggestion
  fun get(): RequestResponseBody? = capture.get()
```

Reply:
Got it! Changed to return a null-safe type.